<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>770</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>971</width>
     <height>731</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>K projection</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>661</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="glLayout"/>
    </widget>
    <widget class="QDial" name="dialTheta">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>130</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="dialPhi">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>130</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="dialRotx">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="dialRoty">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>40</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="dialRotz">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>40</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="showAngles">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Angles</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Fourier</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="glLayoutFourier"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
